*** guile/g-wrap-2.0-guile.pc.in.orig	Sat Dec 27 15:13:27 2008
--- guile/g-wrap-2.0-guile.pc.in	Sat Dec 27 15:13:36 2008
***************
*** 11,15 ****
  Name: G-Wrap
  Description: A Guile wrapper generator
  Version: @VERSION@
! Libs: -L${libdir} -lgwrap-guile-runtime -lgwrap-core-runtime @GUILE_LDFLAGS@ @LIBFFI_LIBS_INSTALLED@
! Cflags: @STD_CFLAGS@ -I${includedir} @GUILE_CFLAGS@ @LIBFFI_CFLAGS_INSTALLED@
--- 11,15 ----
  Name: G-Wrap
  Description: A Guile wrapper generator
  Version: @VERSION@
! Libs: -L${libdir} -lgwrap-guile-runtime -lgwrap-core-runtime @GUILE_LDFLAGS@ @LIBFFI_LIBS@
! Cflags: @STD_CFLAGS@ -I${includedir} @GUILE_CFLAGS@ @LIBFFI_CFLAGS@
